### PR TITLE
working on

### PR DESCRIPTION
**Pull request by tadesse123**
- Project Structure
-Typechecking with Prop Types

### CODE REVIEW

File: src/component/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import InputWithLabel from './InputWithLabel';
+// import './AddTodoForm.css'
```
Comment (kaytiwiita) [inline src/component/AddTodoForm.js:3]: remove unused imports

File: src/component/AddTodoForm.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        console.log(event);
+    };
+
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        if (todoTitle === "") {
+            return false;
+        }
+console.log(todoTitle)
+        // onAddTodo({ fields: { Title: todoTitle }, id: Date.now() });
```
Comment (kaytiwiita) [inline src/component/AddTodoForm.js]: remove unused code

**Review by kaytiwiita — COMMENTED** [review]
Please refer to my previous notes about removing unused lines of code and proper indentation in css files. I understand you committed these changes after I made comments so if there's anything further from the previous comments that's unclear, please reach out.

**Comment by tadesse123** [comment]
Thank you so much for taking time to see my work and giving me a
suggestion. I like the way you look at my work and see what I have to fix.
at the last I was kind of busy I start working and didn't  get enough time
to study. I keep forward to finish this class that is why am kind of way
behind of other students. again thank you so much for your help I
appreciate your effort to help me.

**Pull request merged**